dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/StaticLinkageChecker.java

```diff
@@ -211,8 +216,23 @@ public static void main(String[] arguments)
     Set<FullyQualifiedMethodSignature> availableMethodsInJars = new HashSet<>();
     SyntheticRepository repository = SyntheticRepository.getInstance(classPath);
 
+    URL[] jarFileUrls = jarFilePaths.stream().map(jarPath -> {
+      try {
+        return jarPath.toUri().toURL();
+      } catch (MalformedURLException ex) {
+        System.err.println("Jar file " + jarPath + " was not converted to URL: " + ex.getMessage());
+        return null;
+      }
+    }).filter(Objects::nonNull).toArray(URL[]::new);
+    URLClassLoader classLoaderFromJars = new URLClassLoader(jarFileUrls);
+
     for (FullyQualifiedMethodSignature methodReference : methodReferences) {
       String className = methodReference.getClassName();
+      if (className.startsWith("java.") || className.startsWith("sun.")
+          || className.startsWith("javax.")) {
```

Comment:
javax classes are not necessarily in the JDK. They may need to be pulled in from outside.

Reply:
Removed "javax" from here